torch/_inductor/ir.py

```diff
@@ -6405,17 +6407,43 @@ def __init__(
             - const_args is: [bias, x_scale, x_zp, o_inv_scale, o_zp,
               fp32_output, unary_attr, unary_scalars, unary_algorithm]
         """
-        self.has_bias = len(inputs) == 5
+        self.has_bias = has_bias
+        self.x_scale_zp_are_tensors = x_scale_zp_are_tensors
         super().__init__(
             layout,
             inputs,
             constant_args,
             None,
-            python_kernel_name="torch.ops.onednn.qlinear_pointwise",
-            cpp_kernel_name="onednn::qlinear_pointwise",
+            python_kernel_name=(
+                "torch.ops.onednn.qlinear_pointwise.tensor"
+                if x_scale_zp_are_tensors
+                else "torch.ops.onednn.qlinear_pointwise.default"
+            ),
+            cpp_kernel_name=(
+                "onednn::qlinear_pointwise.tensor"
+                if x_scale_zp_are_tensors
+                else "onednn::qlinear_pointwise.default"
+            ),
         )
         self.cpp_kernel_key = "qlinear_pointwise"
-        self.cpp_op_schema = """
+        self.cpp_op_schema = (
+            """
+            at::Tensor(
+                at::Tensor act,
+                at::Tensor act_scale,
+                at::Tensor act_zero_point,
+                at::Tensor weight,
+                at::Tensor weight_scales,
+                at::Tensor weight_zero_points,
+                c10::optional<at::Tensor> bias,
+                double inv_output_scale,
+                int64_t output_zero_point,
+                c10::optional<c10::ScalarType> output_dtype,
+                std::string post_op_name,
+                torch::List<c10::optional<at::Scalar>> post_op_args,
+                std::string post_op_algorithm)"""
+            if x_scale_zp_are_tensors
```

Comment:
Can we format the string since these 2 op schemas are same for most of the parts?

Reply:
Thanks. It's fixed.